test/test.cc

```diff
@@ -987,6 +1012,7 @@ TEST(RoutingHandlerTest, PreRoutingHandler) {
   {
     Client cli(HOST, PORT);
 
+
```

Comment:
Is this space something intentional?

Reply:
Nope, got borked with the merge conflict